minefana-bungeecord/src/main/java/com/github/games647/minefana/MinefanaBungee.java

```diff
@@ -18,7 +18,7 @@
 
 public class MinefanaBungee extends Plugin implements AnalyticsPlugin {
 
-    private final Logger logger = LoggerFactory.getLogger(getDescription().getName());
+    private final Logger logger = LoggerFactory.getLogger(this.getClass().getName());
```

Comment:
This doesn't work? It should actually use the plugin name here and not the class name.

Reply:
not sure maybe my setup is wrong but without this i get this error https://pastebin.com/sAB0VJaE i am using latest bungee and no plugins on the test server